table/server/common/src/main/java/alluxio/master/table/DatabaseInfo.java

```diff
@@ -47,7 +49,11 @@ public DatabaseInfo(String location, String ownerName, PrincipalType ownerType,
     mOwnerName = ownerName;
     mOwnerType = ownerType;
     mComment = comment;
-    mParameters = params;
```

Comment:
Are `location` and `comment` also need a non-null value? Glue doesn't require user to set values for these fields in the database?

Reply:
Not sure, but those fields were already annotated to be `@Nullable` so I am assuming it is ok to be null.